### PR TITLE
Drop - Deploy a local Ubuntu server hosted with Vagrant and connect it to Azure Arc

### PR DESCRIPTION
**Pull request by mrhoads**
this PR converts the Arc-enabled Server scenario for [deploying a local Ubuntu server hosted with Vagrant and connect it to Azure Arc](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vagrant/local_vagrant_ubuntu)

It's updated to work with multiple providers (e.g. Hyper-V, VirtualBox, libvirt, etc.)

### CODE REVIEW

File: drops/arc_server_deploy_vagrant.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "Type": "tutorial_guide",
+    "Difficulty": "Beginner",
+    "ProgrammingLanguage": [
+        "Bash"
+    ],
+    "Products": [
+      "Arc",
+      "Arc-enabled Servers"
+    ],
+    "LastModified": "",
```
Comment (fcabrera23) [inline drops/arc_server_deploy_vagrant.json:21]: Ensure to add LastModified and CreatedDate dates

File: drops/arc_server_deploy_vagrant.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "Link": "https://www.linkedin.com/in/mike-rhoads/"
+      }
+    ],
+    "Source": "https://github.com/Azure/arc_jumpstart_drops/tree/main/tutorial_guide/deploy_ubuntu_server_vagrant_connect_azure_arc",
+    "Type": "tutorial_guide",
+    "Difficulty": "Beginner",
+    "ProgrammingLanguage": [
+        "Bash"
+    ],
+    "Products": [
```
Comment (fcabrera23) [inline drops/arc_server_deploy_vagrant.json:17]: Maybe add ubuntu?

File: drops/arc_server_deploy_vagrant.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "Title": "Deploy an Ubuntu 22.04 server with Vagrant",
+    "Summary": "This Drop provides a guide to creating an Ubuntu 22.04 server with Vagrant and connecting it to Azure Arc.",
+    "Description": "This Jumpstart Drop provides a step-by-step guide to create an Ubuntu 22.04 server with Vagrant and connecting it to Azure Arc.",
+    "Authors": [
+      {
+        "Name": "Mike Rhoads",
```
Comment (fcabrera23) [inline drops/arc_server_deploy_vagrant.json:7]: Who was the original owner? Maybe we could add him/her?

File: drops/arc_server_deploy_vagrant.json (hunk 2 above)
Comment (fcabrera23) [reply drops/arc_server_deploy_vagrant.json:17]: Maybe also add Vagrant?

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
@mrhoads  could you please review?

**Comment by mrhoads** [comment]
I added:

- Ubuntu and Vagrant as products
- timestamp for LastModified and CreatedDate
- changed Author section to Lior Kamrat based on Git history

**Review by fcabrera23 — APPROVED** [review]
LGTM

**Pull request merged**